### PR TITLE
[VA-17103] Add VBA Operating Status Component

### PR DESCRIPTION
**Pull request by maxx1128**
## Summary

VBA Facility pages have operating statuses that don't use expandable components, which link to a 404 page. This replaces it with an Expandable Alert component with additional info set for the facility.

## Related issue(s)

- _Link to ticket created in va.gov-cms repo_
[department-of-veterans-affairs/va.gov-cms#17103](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/17103)

## Testing done

Locally on tugboat environment, see below screenshots from this URL: https://web-txcrk8bym4a0ivw3xfr8xvdhezmls36f.demo.cms.va.gov/cleveland-va-regional-benefit-office/locations/cleveland-va-regional-benefit-office/#location-and-contact-informati

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| Mobile  |    Screen Shot 2024-02-21 at 4 26 35 PM    |    Screen Shot 2024-02-21 at 4 25 48 PM   |
| Desktop |    Screen Shot 2024-02-21 at 4 26 22 PM    |    Screen Shot 2024-02-21 at 4 25 22 PM   |

## What areas of the site does it impact?

VBA Facility pages

## Acceptance criteria

- [x] The expandable alert components function as expected with the expected information

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [x] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [x] Events are being sent to the appropriate logging solution
- [x] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [x] Did you login to a local build and verify all authenticated routes work as expected with a test user

### CODE REVIEW

**Comment by maxx1128** [comment]
Requesting an accessibility review from @laflannery here.

File: src/site/includes/vba_facilities/expandable_alert.liquid [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  {% assign VBAExpandableAlertStatus = 'warning' %}
+{% elsif fieldOperatingStatusFacility == 'closed' %}
+  {% assign showVBAExpandableAlert = true %}
+  {% assign VBAExpandableAlertTrigger = 'Facility Closed' %}
+  {% assign VBAExpandableAlertStatus = 'error' %}
+{% endif %}
+
+{% if showVBAExpandableAlert %}
+  <va-alert-expandable
+    data-template="components/vba_facilities//expandable-alert.drupal.liquid"
```
Comment (eselkin) [inline src/site/includes/vba_facilities/expandable_alert.liquid:19]: should there be a double slash here?

**Review by eselkin — COMMENTED** [review]
Question about a double slash.

File: src/site/includes/vba_facilities/expandable_alert.liquid (hunk 1 above)
Comment (maxx1128) [reply src/site/includes/vba_facilities/expandable_alert.liquid:19]: That's an extra slash, good catch 👍

**Comment by maxx1128** [comment]
@laflannery I'm getting that same error on the web and CMS side of my tugboat, it looks like there's some server issue that's keeping it from loading.

**Review by laflannery — APPROVED** [review]
This finally loaded for me, Approved!

**Review by eselkin — APPROVED** [review]
LGTM

**Review by chriskim2311 — APPROVED** [review]
LGTM

**Pull request merged**